backend/src/Data/Institution.cs

```diff
@@ -102,8 +102,13 @@ public void Verify()
         State = InstitutionState.VERIFIED;
     }
 
-    public void ChangeOperatingState()
+    public void SwitchOperatingState(InstitutionOperatingState currentState)
     {
-        OperatingState = InstitutionOperatingState.NOT_OPERATING;
+        if(currentState == InstitutionOperatingState.NOT_OPERATING){
+            OperatingState = InstitutionOperatingState.OPERATING;
+        }
+        else{
+            OperatingState = InstitutionOperatingState.NOT_OPERATING;
+        }
```

Comment:
I don't get this logic. For me, the instance variable `OperatingState` of the institution **is** the current operating state of the institution. If we only want to switch (or toggle) the current state, then there is no variable `currentState` needed (as we already know the current state, namely `OperatingState`). In the line https://github.com/building-envelope-data/metabase/pull/183/files/e8d461fc2a04d619093a0412432cf0e7102b0aed..023d0392218cbc1044c8bc4300a677ec2ae71829#diff-866513b90e421e0e02b42b3bb487541d42561d9d1ff1be24510222e58c2c6eefR391 you actually just pass `institution.OperatingState` as `currentState`. What is the purpose of that? What could be given as parameter though is the new state, which I would call `newState`. And then the body of the method just contained one line `OperatingState = newState`, that is, it would be a simple setter method. Personally, I would just remove the `currentState` parameter and replace its usage in the method body simply by  `OperatingState` and remove the argument `institution.OperatingState` in https://github.com/building-envelope-data/metabase/pull/183/files/e8d461fc2a04d619093a0412432cf0e7102b0aed..023d0392218cbc1044c8bc4300a677ec2ae71829#diff-866513b90e421e0e02b42b3bb487541d42561d9d1ff1be24510222e58c2c6eefR391